utils/rpms/daos.spec

```diff
@@ -430,6 +430,8 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # set daos_server to be setgid daos_server in order to invoke daos_server_helper
 # and/or daos_firmware_helper
 %attr(2755,root,daos_server) %{_bindir}/daos_server
+# set daos_metrics to be setgid daos_server to allow attchment to shmem segment
+%attr(2755,root,daos_server) %{_bindir}/daos_metrics
```

Comment:
The reason this was not caught in functional testing is the command is explicitly run with sudo here: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils.py#L1186.  To verify this change we should remove change that line to `command="{} -S {} --csv".format(daos_metrics_exe, engine))` and run with the `test_basiccheckout_sanity` test (`Test-tag: pr test_basiccheckout_sanity`)

Reply:
i'm not familiar with the testing environment and not comfortable fixing that myself. 
@phender if you want to push a change for this to this PR, that would be perfect.
else I'd prefer to get the actual fix landed ASAP (for 2.6.0), and work on the test improvement in another PR (for 2.6.1/2.8).

Reply:
We can address it in a follow on PR.